### PR TITLE
OpenTimelineIO technical charter for review

### PR DESCRIPTION
**Pull request by jmertic**
The template is based on the technical charter template recommended by the ASWF TAC and LF Projects, LLC ( https://github.com/AcademySoftwareFoundation/tac/blob/master/process/charter_template.md ) with the following changes:

- Project name set to `OpenTimelineIO` in all relevant places.
- Mission set based on verbiage in the repo ( Section 1a )
- Inbound project license set to Apache 2
- Both CLA and DCO contribution selection kept
- Outbound project license under a choice of Apache 2 or MIT for any code repositories deemed by the TAC as a 'template' repository.

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

File: project_charters/opentimelineio-charter.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  * **b.** Except as described in Section 7.c., all code contributions to the Project are subject to the following:
+    * **i.** All new inbound code contributions to the Project must be made using the **[Apache License, Version 2.0 (available here: https://www.apache.org/licenses/LICENSE-2.0)]** (the "Project License").
+    * **ii.** All new inbound code contributions must:
+      1. be made pursuant to a duly executed Project Contribution License Agreement (the "CLA"), available on the Project's web site; and
+      2. be accompanied by a Developer Certificate of Origin (http://developercertificate.org) sign-off in the source code system that is submitted through a TSC-approved contribution process which will bind the authorized contributor and, if not self-employed, their employer to the applicable license;
+    * **iii.** All outbound code will be made available under the Project License, with the exception of code repositories designated by the TSC to be used as a template repository, which will be made available under a choice of either the Project License or the MIT License ( available here: https://spdx.org/licenses/MIT.html )
```
Comment (lgritz) [inline project_charters/opentimelineio-charter.md:60]: What does this mean?

Comment (jmertic) [reply project_charters/opentimelineio-charter.md:60]: The project is creating a template for those in the community that wish to develop plugins, and doesn't want to enforce a specific license on those plugins. By providing an option on the license, with one option being MIT, that gives the downstream community as much latitude as possible to relicense as appropriate.

Let me know if that answers your question.

Comment (lgritz) [reply project_charters/opentimelineio-charter.md:60]: Yes, thanks, I just didn't understand what "template repository" was referring to.

**Comment by jminor** [comment]
Ah, thanks for this @jmertic  - we have been iterating on the proposed OTIO TSC charter here: https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/897/files as I didn't realize that it is meant to be hosted here. Should it be in both places, or just one?

**Comment by jmertic** [comment]
@jminor We put the final here, and then you are welcome to put in your repo as well or reference it here.. Generally we like to have them centralized to help folks find them easily.

**Comment by jminor** [comment]
@jmertic could you close this PR in favor of this one that I just submitted?
https://github.com/AcademySoftwareFoundation/foundation/pull/6

That will allow me to make changes as we iterate on the (hopefully) last details.

**Pull request closed without merging**